expirationd-scm-1.rockspec

```diff
@@ -18,6 +18,7 @@ build = {
     type = "builtin",
     modules = {
         ["expirationd"] = "expirationd.lua",
+        ["expirationd-version"] = "expirationd-version.lua",
```

Comment:
I think it's worth getting away from the flat structure if we have more than one module file:

```bash
$ mkdir expirationd
$ git mv expiradiond.lua expirationd/init.lua
$ git mv expirationd-version.lua expirationd/version.lua 
```